src/frontends/onnx/frontend/src/op/com.microsoft/range.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/com.microsoft/range.hpp"
+
+#include <memory>
+
+#include "default_opset.hpp"
+
+OPENVINO_SUPPRESS_DEPRECATED_START
+namespace ngraph {
```

Comment:
ONNX FE has been slightly refactored.
Please use the following files of onnx frontend to remove usage of ngraph and use OV

[/src/frontends/onnx/frontend/src/op/leaky_relu.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/leaky_relu.hpp)
[/src/frontends/onnx/frontend/src/op/leaky_relu.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/leaky_relu.cpp)